File: corp-obl-backend/src/main/java/com/tx/co/back_office/tasktemplate/api/model/ObjectSearchTaskTemplate.java
package com.tx.co.back_office.tasktemplate.api.model;

import java.util.List;

import com.tx.co.back_office.company.domain.Company;
import com.tx.co.back_office.topic.domain.Topic;

public class ObjectSearchTaskTemplate {

	private String descriptionTaskTemplate;
	List<Company> companies;
	List<Topic> topics;
	
	public String getDescriptionTaskTemplate() {
		return descriptionTaskTemplate;
	}
	public void setDescriptionTaskTemplate(String descriptionTaskTemplate) {
		this.descriptionTaskTemplate = descriptionTaskTemplate;
	}
	public List<Company> getCompanies() {
		return companies;
	}
	public void setCompanies(List<Company> companies) {
		this.companies = companies;
	}
	public List<Topic> getTopics() {
		return topics;
	}
	public void setTopics(List<Topic> topics) {
		this.topics = topics;
	}
	
	
}

File: corp-obl-frontend/src/app/back_office/tasktemplate/model/object-search-tasktemplate.ts
import {Company} from '../../company/model/company';
import {Topic} from '../../topic/model/topic';

export class ObjectSearchTaskTemplate {

    descriptionTaskTemplate: string;
    companies: Company[];
    topics: Topic[];

    constructor(descriptionTaskTemplate: string, companies: Company[], topics: Topic[]) {
        this.descriptionTaskTemplate = descriptionTaskTemplate;
        this.companies = companies;
        this.topics = topics;
    }
}
